unittests/subjective_billing_tests.cpp

```diff
@@ -96,9 +98,9 @@ BOOST_AUTO_TEST_CASE( subjective_bill_test ) {
    { // expired handling logic, full billing until expiration then failed/decay logic
       subjective_billing sub_bill;
 
-      sub_bill.subjective_bill( id1, now, a, fc::microseconds( 1024 ) );
-      sub_bill.subjective_bill( id2, now + fc::microseconds(1), a, fc::microseconds( 1024 ) );
```

Comment:
This test was a bit nonsensical as a trx expiration is always to a second as it is a `time_point_sec`. Updated to use `fc::second` as that is more valid.